src/main/java/seedu/address/logic/parser/MailXCommandParser.java

```diff
@@ -24,26 +25,41 @@
     public MailXCommand parse(String args) throws ParseException {
         requireNonNull(args);
         ArgumentMultimap argMultimap =
-                ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_EMAIL, PREFIX_MOD, PREFIX_GROUP);
+                ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_EMAIL,
+                        PREFIX_MOD, PREFIX_GROUP, PREFIX_STUDENT_NUMBER);
 
-        MailMDescriptor mailMDescriptor = new MailMDescriptor();
+        System.out.println(argMultimap.getPreamble());
```

Comment:
Should this be deleted? I don't think there should be any printlines right?

Reply:
Yeah, thank you for pointing out.